src/transformers/models/detr/configuration_detr.py

```diff
@@ -169,6 +179,17 @@ def __init__(
         eos_coefficient=0.1,
         **kwargs
     ):
+        if not use_timm_backbone:
+            if backbone_config is None:
+                logger.info("`backbone_config` is `None`. Initializing the config with the default `ResNet` backbone.")
+                backbone_config = ResNetConfig(out_features=["stage2", "stage3", "stage4"])
```

Comment:
Like in the other PR, consider using `CONFIG_MAPPING["resnet"]` here to avoid the dependency on the resnet model.